backend/data/processors/manual_fixes/AEROFixes.py

```diff
@@ -43,7 +41,7 @@ def AERO_4110():
         "processed": "At least 144 Units in AEROAH stream. AERO3110"
     """
 
-    return "144UOC in AEROAH && AERO3110"
+    return "144UOC && AEROAH && AERO3110"
```

Comment:
```suggestion
    return "144UOC in AEROAH && AERO3110"
```
the original manual fix was correct - this course requires completing 144uoc of courses that are in the AEROAH stream, not completing 144uoc of courses total

Reply:
oh strange, I didn't change any of the manual fixes files